website/docs/Pipelines/RNA_with_UMIs_Pipeline/rna-with-umis.methods.md

```diff
@@ -0,0 +1,23 @@
+
+Following alignment, the read group information from the genome-aligned BAM was copied to the transcriptome-aligned BAM using Samtools. Both BAM files were then sorted by coordinate with Picard's SortSam tool, and index and MD5 digest files were created. Duplicate reads were sorted by UMI with UMI-tools to differentiate between biological and PCR duplicates. To specify the tag where the UMIs were stored, --extract-umi-method was set to "tag" and --umi-tag was set to "RX". Unmapped reads were deduplicated with --unmapped-reads = use. Tagged BAM files were output using the option --output-bam. Picard's SortSam tool was used again to sort the BAM files by queryname and create MD5 digest files. PCR duplicates were marked, and duplicate metrics were calculated using Picard's MarkDuplicates tool with GATK v4.1.9.0. BAM files were sorted with Picard's SortSam tool, and index and MD5 digest files were created.
+
+The genome-aligned, duplicate-marked BAM file was then used to calculate summary metrics using RNASeQC, Picard's CollectRNASeqMetrics and CollectMultipleMetrics tools, and GATK's (v4.2.5.0) GetPileupSummaries and CalculateContamination tools. CollectMultipleMetrics was used with the programs “CollectInsertSizeMetrics” and “CollectAlignmentSummaryMetrics”. GetPileupSummaries was run with the read filters, "WellformedReadFilter" and "MappingQualityAvailableReadFilter" disabled.
```

Comment:
"GetPileupSummaries was run with the read filters, "WellformedReadFilter" and "MappingQualityAvailableReadFilter" disabled."—probably another implementation detail we can omit here. Although first I should ask, who's the intended reader?

Reply:
"GATK's GetPileupSummaries and CalculateContamination tools" -> "GATK's CalculateContamination." GetPileupSummaries is just a preprocessing step.

Reply:
I hesitate to remove GetPileupSummaries - to me it seems like an important detail for getting the same results.

Reply:
Yea actually the read filters for GetPileupSummaries are important, so good call!